react-native/react/native/notifications.desktop.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+/* @flow */
+
+import engine from '../engine'
+
+const param = {
+  channels: {
+    session: true,
+    users: true
+  }
+}
+
+export function enableNotifications () {
+  console.log('setting up notification')
+  engine.listenOnConnect(() => {
+    engine.rpc('notifyCtl.toggleNotifications', param, {}, (error, response) => {
+      if (error != null) {
+        console.error('error in toggling notifications: ', error)
+      }
+      console.log('Enabled Notifications')
+    })
+  })
+}
+
+export function bindNotifications () {
```

Comment:
maybe we should call a helper that'll do this automatically so we don't need super similar code here and in the unbind?